setup.py

```diff
@@ -38,7 +38,7 @@ def find_version(*file_paths):
     include_package_data=True,
     zip_safe=True,
     python_requires=">=3.6",
-    install_requires=["cloudformation-cli>=0.1,<0.2", "docker>=3.7,<5"],
+    install_requires=["cloudformation-cli>=0.1.10,<0.2", "docker>=3.7,<5"],
```

Comment:
Is there a way to make this generic? Or is it a best practice to update this every time cloudformation-cli is updated?

Reply:
this isn't necessary every time we update, but older versions of the cloudformation-cli don't work with this plugin as they provide things like the partition, which hard fails the python plugin lib when we try to run contract tests. This isn't necessary every time, just a one off because anything before 0.1.9 isn't compatible.